manifests/init.pp

```diff
@@ -331,7 +331,7 @@
   String $user                                    = $jenkins::params::user,
   Boolean $manage_group                           = $jenkins::params::manage_group,
   String $group                                   = $jenkins::params::group,
-  Array $default_plugins                          = $jenkins::params::default_plugins,
```

Comment:
This makes it a breaking change. Are you sure we want that?

Reply:
How would you explicitly set the version for the plugins in your current setup? It seems like with an array you only set the plugins but with the limitation that that you only get the latest

Reply:
```puppet
$my_plugins = {
  'credentials' => { 'version' => 'x.y.z' },
  'structs'     =>  { 'version' => 'a.b.c' },
}
class { 'jenkins': 
  default_plugins => [],
  plugin_hash     => $my_plugins,
}
```

Reply:
Ah fair enough. That was an oversight on my part and I'll do that instead! If you could put that in the documentation that would be very useful. Will close the issue + PR